src/Resources/views/CRUD/Association/edit_one_to_many_inline_table.html.twig

```diff
@@ -23,8 +23,8 @@ file that was distributed with this source code.
                             style="display:none;"
                         {% endif %}
                     >
-                        {{ nested_field.vars.label|trans({}, nested_field.vars['sonata_admin'].admin.translationDomain
-                            |default(nested_field.vars.translation_domain)
+                        {{ nested_field.vars.label|trans({}, nested_field.vars.translation_domain
+                            |default(nested_field.vars['sonata_admin'].admin.translationDomain)
                         ) }}
```

Comment:
Shouldn't we also support false, with something like ?

```suggestion
    if (false is same as nested_field.vars.translation_domain) {
        {{ nested_field.vars.label }}
    } else {
        {{ nested_field.vars.label|trans({}, 
            nested_field.vars.translation_domain|default(nested_field.vars['sonata_admin'].admin.translationDomain)
        ) }}
```

Reply:
Sure, done !